packages/manager/modules/bm-server-components/src/netboot/translations/Messages_fr_FR.json

```
@@ -38,5 +39,10 @@
   "server_configuration_netboot_confirm": "Valider",
   "server_configuration_netboot_review": "Résumé",
   "server_configuration_netboot_save_in_progress": "Mise à jour...",
-  "server_configuration_netboot_invalid": "Invalide"
+  "server_configuration_netboot_invalid": "Invalide",
+  "server_configuration_netboot_auth_method": "Méthode d'authentification :",
+  "server_configuration_netboot_auth_method_sshkey": "Authentification par clé SSH",
+  "server_configuration_netboot_auth_method_password": "Authentification par mot de passe",
+  "server_configuration_netboot_sshkey_input": "Votre clé SSH Publique :",
+  "server_configuration_netboot_sshkey_input_placeholder": "Insérez une valeur de type sshPubKey. Exemple : ssh-rsa AAAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX== my-public-key"
```

@Dampen59 `ssh-rsa AAAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX== my-public-key` should be in packages/manager/modules/bm-server-components/src/netboot/constants.js
And translation should contains a variable, so  `ssh-rsa AAAAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX== my-public-key` is passed as argument to the translation
```suggestion
  "server_configuration_netboot_sshkey_input_placeholder": "Insérez une valeur de type sshPubKey. Exemple : {{t0}}"
```